tsconfig.json

```diff
@@ -66,8 +66,7 @@
     },
     "typeRoots": [
       "./node_modules/@polkadot/ts",
-      "./node_modules/@types",
-      "./node_modules/@joystream/types/lib"
+      "./node_modules/@types"
```

Comment:
I noticed tsconfig has a target of esnext, I think we should be a bit more conservative and lower it to something lower that is guaranteed to work with node v12 LTS rather than v14.

Reply:
this would also apply to the types library as well but it will have to wait for the next release